code-challenges.test.js

```diff
@@ -50,7 +105,34 @@ const hand1 = [5, 5, 5, 3, 3]
 const hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
+// // Expected output: false
+
+describe("theFull", () => {
+    it("takes in an array of 5 numbers and determines whether or not the array is a “full house.", () => {
+        const hand1 = [5, 5, 5, 3, 3]
+        const hand2 = [5, 5, 3, 3, 4]
+        const hand3 = [5, 5, 5, 5, 4]
+      expect(theFull(hand1)).toEqual(expect.arrayContaining([true]))
+      expect(theFull(hand2)).toEqual(expect.arrayContaining([false]))
+      expect(theFull(hand3)).toEqual(expect.arrayContaining([false]))
+    })
+  })
 
+  //     ReferenceError: theFull is not defined
 
 // b) Create the function that makes the test pass.
+
+// create function that takes in array of 5 numbers
+// have it determin if the array contains 2 same numbers and 3 same numbers
+// if both of those are true than it is true that it is a full house.
+// if one or both is flase than it is not a full house
+
+const theFull = (arr) => {
+    if arr.inculdes()
+    return true
+    if arr.includes()
+    return true
+    if true
+    return full house
+    else not full house
+console.log(theFull)
```

Comment:
.includes is a useful tool. But it's not quite the tool for the job here. Why not trace the flow of data at each line and watch as the data gets updated or transformed. 

All functions do is take in data and transform it to some kind of output.